amoro-ams/amoro-ams-server/pom.xml

```diff
@@ -310,14 +310,14 @@
         <dependency>
             <groupId>org.apache.paimon</groupId>
             <artifactId>paimon-s3</artifactId>
-            <scope>runtime</scope>
+            <scope>${paimon-dependency-scope}</scope>
```

Comment:
I'm not sure if we need to emphasize the scope here. Perhaps it can be inherited directly from the parent?